virttest/qemu_monitor.py

```diff
@@ -1939,20 +1938,18 @@ def get_backingfile(self, device):
             pass
         return backing_file
 
-    def block_mirror(self, device, target, speed, sync, format, mode,
-                     cmd="drive-mirror", correct=True):
+    def block_mirror(self, device, target, sync, cmd="drive-mirror",
+                     correct=True, **kwargs):
         """
         Start mirror type block device copy job
 
-        :param device: device ID
-        :param target: target image
-        :param speed: limited speed, unit is B/s
+        :param device: device name to operate on
+        :param target: name of new image file
         :param sync: what parts of the disk image should be copied to the
-                     destination;
-        :param mode: 'absolute-paths' or 'existing'
-        :param format: target image format
+                     destination
         :param cmd: block mirror command
         :param correct: auto correct command, correct by default
+        :param kwargs: optional keyword arguments
 
         :return: The command's output
```

Comment:
Frankly speaking,
I personally don't like to use **kwargs,  especially for API function to tp-qemu.
It's hard to understand what and how many params needed.
In contrast, The old code and its' doc string  looks more clear for the caller.